hmda-data-publisher/src/test/scala/hmda/publisher/helper/DateGeneratorSpec.scala

```diff
@@ -7,11 +7,11 @@ class DateGeneratorSpec extends WordSpec with Matchers {
   "date generator" should {
 
     "return correct current date" in {
-      assert(generator.currentDate == "2024-06-19-")
+      assert(generator.currentDate == "2024-06-25-")
     }
 
     "return correct current quarterly date" in {
-      assert(generator.currentQuarterlyDate == "2024-06-19_")
+      assert(generator.currentQuarterlyDate == "2024-06-25_")
```

Comment:
Hmm. Does this work if we run it tomorrow? :-)
Do we need some kind of alternative current date string generating resource here?